Lecture2/Task2/Task2L2.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
+
+public class Task2L2 {
+
+    private static final String MESSAGE_SELECTED = "You have selected a ";
+    private static final String MESSAGE_INVALID = "You have entered an invalid day's index";
+
+    public static void main ( String[] args ) {
+        Scanner in = new Scanner ( System.in );
```

Comment:
fix formatting. You don't need spaces before and in brackets, i.e. should be:
Scanner in = new Scanner(System.in);
The same is for day = in.nextByte ( ); and below

Reply:
fixed